src/plugin/regions.js

```diff
@@ -639,12 +780,25 @@ export default class RegionsPlugin {
                 region = this.add(params || {});
             }
 
```

Comment:
Not sure if this really needs to be updated on each interaction - but moved it to `eventDown` so it is not being called on every mouse move.